apps/mull-api/src/app/auth/facebook.strategy.ts

```diff
@@ -0,0 +1,37 @@
+import { PassportStrategy } from '@nestjs/passport';
+import { Profile, Strategy, StrategyOption } from 'passport-facebook';
+import { environment } from '../../environments/environment';
+import { Injectable } from '@nestjs/common';
+import { User } from '../entities';
+import { AuthService } from './auth.service';
+import { RegistrationMethod } from '@mull/types';
+
+@Injectable()
+export class FacebookStrategy extends PassportStrategy(Strategy, 'facebook') {
+  constructor(private authService: AuthService) {
+    super({
+      clientID: environment.auth.facebook.clientId,
+      clientSecret: environment.auth.facebook.clientSecret,
+      callbackURL: 'http://localhost:3333/api/auth/facebook/redirect',
```

Comment:
How is this gonna work when the application is deployed? Might need to put this in the environment.ts file

Reply:
Right, I could add a BASE_URL variable

Reply:
this can be done in #30, just wanted to mention it